conf.py

```diff
@@ -10,9 +10,9 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 #
-# import os
-# import sys
-# sys.path.insert(0, os.path.abspath('.'))
+import os
+import sys
+sys.path.insert(0, os.path.abspath('../improve/framework.py'))
```

Comment:
Could you wrap this in an IF statement? Check if the directory exists? What happens if there is no improve dir?